assets/js/theme/common/carousel/index.js

```diff
@@ -55,8 +83,8 @@ export default function () {
         // getting element using find to pass jest test
         const $carousel = $(document).find(carousel);
 
-        $carousel.on('init', onCarouselChange);
-        $carousel.on('afterChange', onCarouselChange);
+        // click event has been added since afterChange doesn't contain isTrusted property
```

Comment:
Could you please explain why do we need isTrusted property?

Reply:
it helps to detect when action was generated by User. Since we need to make announcements only on User's actions.

Reply:
What kind of non user interaction could lead to handler call?